monai/handlers/checkpoint_saver.py

```diff
@@ -37,6 +37,9 @@ class CheckpointSaver:
         name: identifier of logging.logger to use, if None, defaulting to ``engine.logger``.
         file_prefix: prefix for the filenames to which objects will be saved.
         save_final: whether to save checkpoint or session at final iteration or exception.
+            if want to save checkpoint when exception happened, please put this handler before
```

Comment:
"if want to save checkpoint when exception happened, please put this handler before..." -> "If checkpoints are to be saved when an exception is raised, put this handler before..."

Reply:
updated in the latest commit.
Thanks.